cli/internal/globwatcher/globwatcher.go

```diff
@@ -9,13 +9,19 @@ import (
 	"github.com/hashicorp/go-hclog"
 	"github.com/vercel/turborepo/cli/internal/doublestar"
 	"github.com/vercel/turborepo/cli/internal/filewatcher"
+	"github.com/vercel/turborepo/cli/internal/fs"
 	"github.com/vercel/turborepo/cli/internal/turbopath"
 	"github.com/vercel/turborepo/cli/internal/util"
 )
 
 // ErrClosed is returned when attempting to get changed globs after glob watching has closed
 var ErrClosed = errors.New("glob watching is closed")
 
+type Globs struct {
```

Comment:
If these aren't used outside this package, you can un-export the struct and the fields.

Note that tests are typically in the same package as code they are testing, so don't require exported identifiers.